### PR TITLE
Add keyboard shuffle feature

### PR DESCRIPTION
**Pull request by Siimone**
- shuffle keyboard with an optional button for more privacy when unlocking wallet in public places
- fix typo

GIF of the first feature: [link](https://imgur.com/a/oNYxifu)

### CODE REVIEW

File: green/src/main/res/layout/green_pin_view.xml [hunk 1]
```diff
@@ -255,6 +259,20 @@
             app:layout_constraintStart_toEndOf="@+id/button8"
             app:layout_constraintTop_toTopOf="@+id/button8" />
 
+        <Button
+            android:id="@+id/buttonDelete2"
```
Comment (lvaccaro) [inline green/src/main/res/layout/green_pin_view.xml:263]: nit: I think you don't need the id in this case

File: green/src/main/java/com/blockstream/green/views/GreenPinView.kt [hunk 2]
```diff
@@ -44,6 +44,35 @@ class GreenPinView @JvmOverloads constructor(
             deletePinDigit(true)
             true
         }
+
+        binding.shuffleListener = OnClickListener { view ->
+            var digitsValues: Array<String> = arrayOf("1", "2", "3", "4", "5", "6", "7", "8", "9", "0")
+            var parentRoot = view.rootView
+            // get all PIN buttons
+            var button0 = parentRoot.findViewById<Button>(R.id.button0);
```
Comment (lvaccaro) [inline green/src/main/java/com/blockstream/green/views/GreenPinView.kt:52]: nit: could you use an array instead 10 button variables? If so, you could assign digitaValues in a single loop

**Comment by lvaccaro** [comment]
Hi @Siimone , IMO that's a really cool feature!

utack f21c8111b15253ed6b677497ce06982987650597 with nits

**Comment by angelix** [comment]
@Siimone included with commit eb831fbb81bde63c06e8ec27f55e52644871629a

**Pull request closed without merging**